src/core/structure/sheet.ts

```diff
@@ -439,7 +436,8 @@ export default class Sheet {
    * Returns true if the value of the cell has changed.
    */
   private resolveCell(cell: Cell, colKey: ColumnKey, rowKey: RowKey): boolean {
-    const evalResult = this.expressionHandler.evaluate(cell.formula);
+    const expressionHandler = new ExpressionHandler(this, cell.formula);
+    const evalResult = expressionHandler.evaluate();
```

Comment:
Great idea to accommodate for the way math.js works :+1: This will have a greater memory footprint, but I doubt it's noticeable until some absurd amount of cells. Maybe after #99 this could be optimized to only initialize for cells with an actual formula.

Reply:
It is just a single object, it will absolutely not matter and even if there is a huge amount of cells, this object is created and destroyed in the evaluation process, it will not use memory it maybe slower by few nanoseconds.

Reply:
My bad, I somehow understood that the ExpressionHandler would persist in the Cell object. Wouldn't that be preferable to initializing the object each time the formula changes?

Reply:
I don't see the benefit of storing the object, the calculations need to be redone anyway. The point of my change is that `ExpressionHandler` is a transient object, you can think of it as a fancy function, hell even functions are objects in JS.